js/viewer.js

```diff
@@ -8,7 +8,8 @@ if (typeof String.prototype.endsWith !== "function")
         return this.indexOf(suffix, this.length - suffix.length) !== -1;
     };
 
-var local = location.href.indexOf('://www.cyberbotics.com/doc') == -1;
+var local = location.href.indexOf('://www.cyberbotics.com/doc') == -1 &&
+            location.href.indexOf('://robotbenchmark.net/webots_documentation') == -1 ;
```

Comment:
It is a pity to hard-code `robotbenchmark.net` here, as we will have to maintain this each time we deploy webots.js on another web site. Can't we find a better solution?